package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-rev",
-  "version": "5.1.0",
+  "version": "5.1.1",
```

Comment:
No, just don't change this at all. We'll release it but that's separate from this PR.